src/bors/handlers/mod.rs

```diff
@@ -122,32 +122,23 @@ pub async fn handle_bors_global_event<Client: RepositoryClient>(
     let db = Arc::clone(&ctx.db);
     match event {
         BorsGlobalEvent::InstallationsChanged => {
-            let span = tracing::info_span!("Repository reload").entered();
-
-            match ctx.repository_loader.load_repositories().await {
-                Ok(reloaded_repos) => {
-                    let mut repositories = ctx.repositories.write().unwrap();
-                    for repo in repositories.values() {
-                        if !reloaded_repos.contains_key(&repo.repository) {
-                            tracing::info!("Repository {} was not reloaded", repo.repository);
-                        }
-                    }
-                    for repo in reloaded_repos.values() {
-                        if repositories
-                            .insert(repo.repository.clone(), repo.clone())
-                            .is_some()
-                        {
-                            tracing::info!("Repository {} was reloaded", repo.repository);
-                        } else {
-                            tracing::info!("Repository {} was added", repo.repository);
-                        }
-                    }
+            let reloaded_repos = ctx.repository_loader.load_repositories().await?;
```

Comment:
We can wrap this whole thing in `async {}` (or even better, create an `async fn`), and then instrument it with a span, so that we keep the span, it would be nice to see that this is happening in logs.